runtime/src/bank.rs

```diff
@@ -1849,6 +1849,41 @@ impl Bank {
         self.epoch_reward_status = EpochRewardStatus::Inactive;
     }
 
+    #[allow(dead_code)]
+    /// Begin the process of calculating and distributing rewards.
+    /// This process can take multiple slots.
+    fn begin_partitioned_rewards(
+        &mut self,
+        parent_epoch: Epoch,
+        reward_calc_tracer: Option<impl Fn(&RewardCalculationEvent) + Send + Sync>,
+        thread_pool: &ThreadPool,
+        parent_slot: Slot,
+        parent_height: u64,
+        rewards_metrics: &mut RewardsMetrics,
+    ) {
+        let CalculateRewardsAndDistributeVoteRewardsResult {
+            stake_rewards_by_partition,
+            ..
+        } = self.calculate_rewards_and_distribute_vote_rewards(
+            parent_epoch,
+            reward_calc_tracer,
+            thread_pool,
+            rewards_metrics,
+        );
+
+        let slot = self.slot();
+        self.set_epoch_reward_status_active(stake_rewards_by_partition);
+
+        datapoint_info!(
+            "reward-status-update",
+            ("slot", slot, i64),
+            ("height", self.block_height(), i64),
+            ("activate", 1, i64),
+            ("parent_start_slot", parent_slot, i64),
+            ("parent_start_height", parent_height, i64),
```

Comment:
ok

Reply:
I think it is valuable to log parent_start_slot/height. It will help us to examine timing in fork situations. 
Since this logging only happens at epoch boundary, they are not expensive.  
I would prefer to log them.